### PR TITLE
Add support for Internal and Private network types on windows

### PR DESCRIPTION
**Pull request by msabansal**
In windows we have supported these two additional types since RS1.

Internal network allows inter container connectivity with support for host/container communication.
Private network allows just inter container connectivity.

This PR enables these two networking modes on windows

### CODE REVIEW

File: drivers_windows.go [hunk 1]
```diff
@@ -16,6 +16,8 @@ func getInitializers(experimental bool) []initializer {
 		{windows.GetInit("l2bridge"), "l2bridge"},
 		{windows.GetInit("l2tunnel"), "l2tunnel"},
 		{windows.GetInit("nat"), "nat"},
+		{windows.GetInit("internal"), "internal"},
+		{windows.GetInit("private"), "naprivatet"},
```
Comment (fcrisciani) [inline drivers_windows.go]: is this supposed to be private?

**Comment by msabansal** [comment]
Yes. But I was able to test this with using -d driver switch. I will fix this.

Sent from my Windows 10 device

________________________________

File: drivers_windows.go (hunk 1 above)
Comment (msabansal) [reply drivers_windows.go]: This worked:
PS C:\Program Files\Debugging Tools for Windows (x64)> docker network ls
NETWORK ID          NAME                DRIVER              SCOPE
be554c4fbc1a        nat                 nat                 local
37a70ad722e5        none                null                local
778c644db3f6        test1               private             local

but I have fixed the typo

**Comment by selansen** [comment]
LGTM

**Comment by arkodg** [comment]
@pradipd is this still needed ?

**Comment by pradipd** [comment]
Yes. Can we please merge this?  Thanks.

**Comment by arkodg** [comment]
thanks @pradipd 

PTAL @euanh

**Comment by arkodg** [comment]
any specific reason to make it experimental ?

To enable this driver as experimental, another PR would be need in the same file `drivers_windows.go`

**Comment by pradipd** [comment]
Ignore that.  It is fine as is.

**Pull request merged**